aten/src/ATen/native/native_functions.yaml

```diff
@@ -3536,6 +3540,7 @@
     SparseCPU: sparse_to_dense
     SparseCUDA: sparse_to_dense
     MkldnnCPU: mkldnn_to_dense
+    Vulkan: vulkan_to_dense
```

Comment:
vulkan_to_dense should be a noop (and similarly to regular Dense it's not even declared here).

unlike mkldnn, vulkan is a device, not layout. So you should just put this transfer logic into copy_